README.md

```diff
@@ -381,18 +390,33 @@ select
 from {{ref('my_model')}}
 ```
 
-#### union_tables ([source](macros/sql/union.sql))
-This macro implements an "outer union." The list of relations provided to this macro will be unioned together, and any columns exclusive to a subset of these tables will be filled with `null` where not present. The `column_override` argument is used to explicitly assign the column type for a set of columns. The `source_column_name` argument is used to change the name of the`_dbt_source_table` field.
+#### union_relations ([source](macros/sql/union.sql))
+> This replaces the `union_tables` macro. Note that the `union_tables` macro will
+be deprecated in a future release of this package.
 
-Usage:
+This macro unions together an array of [Relations](https://docs.getdbt.com/docs/api-variable#section-relation),
+even when columns have differing orders in each Relation, and/or some columns are
+missing from some relations. Any columns exclusive to a subset of these
+relations will be filled with `null` where not present. An new column
+(`_dbt_source_relation`) is also added to indicate the source for each record.
+
+**Usage:**
 ```
-{{ dbt_utils.union_tables(
-    tables=[ref('table_1'), ref('table_2')],
-    column_override={"some_field": "varchar(100)"},
-    exclude=["some_other_field"],
-    source_column_name='custom_source_column_name'
+{{ dbt_utils.union_relations(
+    relations=[ref('my_model'), source('my_source', 'my_table')],
+    exclude=["_loaded_at"]
 ) }}
 ```
+**Args:**
+* `relations` (required): An array of [Relations](https://docs.getdbt.com/docs/api-variable#section-relation).
+* `exclude` (optional): A list of column names that should be excluded from
+the final query.
+* `include` (optional): A list of column names that should be included in the
```

Comment:
I'd maybe change "blacklist/whitelist" to something like `the "exclude" and "include" parameters are mutually exclusive" or something like that